.github/patch.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+register_shutdown_function(function () {
+    $status = opcache_get_status(false);
+    var_dump($status);
+
+    $ok = true;
+    if ($status["memory_usage"]["free_memory"] < 10*1024*1024) {
```

Comment:
I'm not really sure how much value this provides. I agree that `opcache.memory_consumption` and `opcache.jit_buffer_size` should be set sufficiently high. However, running out of shm is a real-world scenario that likely happens more than it should. Having this case tested may be considered useful.

Reply:
Removed the die; then :)